pkg/k8s/camel.go

```diff
@@ -0,0 +1,70 @@
+package k8s
+
+import (
+	"regexp"
+	"strings"
+)
+
+var numberSequence = regexp.MustCompile(`([a-zA-Z])(\d+)([a-zA-Z]?)`)
+var numberReplacement = []byte(`$1 $2 $3`)
+
+func addWordBoundariesToNumbers(s string) string {
+	b := []byte(s)
+	b = numberSequence.ReplaceAll(b, numberReplacement)
+	return string(b)
+}
+
+// Converts a string to CamelCase
+func toCamelInitCase(s string, initCase bool) string {
+	s = addWordBoundariesToNumbers(s)
+	s = strings.Trim(s, " ")
+	n := ""
+	capNext := initCase
+	for _, v := range s {
+		if v >= 'A' && v <= 'Z' {
+			n += string(v)
+		}
+		if v >= '0' && v <= '9' {
+			n += string(v)
+		}
+		if v >= 'a' && v <= 'z' {
+			if capNext {
+				n += strings.ToUpper(string(v))
+			} else {
+				n += string(v)
+			}
+		}
+		if v == '_' || v == ' ' || v == '-' || v == '.' {
+			capNext = true
+		} else {
+			capNext = false
+		}
+	}
+	return n
+}
+
+var uppercaseAcronym = map[string]bool{
```

Comment:
Why have you copied strcase lib? :)

Reply:
Because we only need one function from the entire library. And it's not worth tying up to an external library for this. As Dave Cheney said, if there is an opportunity to copy a small thing, it is better to copy it than to organize a new connection.